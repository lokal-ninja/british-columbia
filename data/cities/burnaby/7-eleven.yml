id: 5847961082
type: node
latitude: 49.22158
longitude: -123.02304
postcode: ""
city: Burnaby
street: Imperial Street
housenumber: "3720"
phone: ""
opening_hours: ""
website: https://stores.7-eleven.ca/store/37858
