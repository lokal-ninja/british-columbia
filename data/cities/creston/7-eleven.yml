id: 367124674
type: node
latitude: 49.09584
longitude: -116.50651
postcode: ""
city: Creston
street: Canyon Street
housenumber: "1608"
phone: ""
opening_hours: ""
website: https://stores.7-eleven.ca/store/20985
