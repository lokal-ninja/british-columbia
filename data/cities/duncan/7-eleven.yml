id: 10579233659
type: node
latitude: 48.78039
longitude: -123.68418
postcode: "V9L 5G6"
city: Duncan
street: "Lakes Road"
housenumber: 400
phone: "+1-250-748-6924"
opening_hours: "24/7"
website: ""