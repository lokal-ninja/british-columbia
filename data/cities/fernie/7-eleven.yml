id: 223981280
type: way
latitude: 49.51153
longitude: -115.06015
postcode: "V0B 1M0"
city: Fernie
street: "BC-3"
housenumber: 1261
phone: "+1-250-423-7662"
opening_hours: ""
website: ""
