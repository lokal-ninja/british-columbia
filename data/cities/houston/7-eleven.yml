id: 409216884
type: way
latitude: 54.39699
longitude: -126.65766
postcode: "V0J 1Z0"
city: Houston
street: "Yellowhead Highway"
housenumber: 3066
phone: ""
opening_hours: ""
website: "https://stores.7-eleven.ca/store/32141"