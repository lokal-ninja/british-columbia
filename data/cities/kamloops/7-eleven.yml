id: 1685269920
type: node
latitude: 50.65654
longitude: -120.34086
postcode: V2E 1E9
city: Kamloops
street: Summit Drive
housenumber: "1590"
phone: +1 250 372 5752
opening_hours: 05:30-23:00
website: https://www.7-eleven.ca/locations/bc/kamloops/1590-summit-drive-1-25748
