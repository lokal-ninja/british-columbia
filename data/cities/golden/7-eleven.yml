id: 166907839
type: node
latitude: 51.3
longitude: -116.96533
postcode: V0A 1H0
city: Golden
street: 9 Avenue North
housenumber: "410"
phone: ""
opening_hours: 24/7
website: https://stores.7-eleven.ca/store/29435
